### DISCUSSION TITLE
The method does not exist / is not available

### DISCUSSION THREAD

**Discussion by Janaka-Steph** [MetaMask/snaps#2395] [Q&A]
I want to trigger a send.
```
 await ethereum.request({
              method: 'eth_sendTransaction',
              params: [
                {
                  to: '0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045',
                  from: '0xFB468D3Cb816A3E167AB532e7eb19C8d42259939',
                  value: '0x16345785d8a0000',
                },
              ],
            });
```
Returns:
`An error happened: The method does not exist / is not available.`

Calling `wallet_getPermissions` just before I can see that the result contains a permission with a parentCapability of `eth_accounts`and correct value.

Permissions:
```
  "initialPermissions": {
    "snap_dialog": {},
    "snap_manageState": {},
    "endowment:ethereum-provider": {},
    "endowment:network-access": {},
    "endowment:rpc": {
      "dapps": true,
      "snaps": false
    }
  }
```

According to https://github.com/MetaMask/snaps/issues/2158 `eth_sendTransaction` is not available.
`eth_signTransaction` is not available either https://github.com/MetaMask/snaps/issues/1420
Is there an other way to make a transaction?

Any idea?

**Comment by Montoya** [comment]
The only option that may be available is `eth_sendRawTransaction` which would require first signing the transaction separately. The other methods are not available as you have seen. 

Where are you trying to do the transaction? Usually it has to be initiated from a dapp, not a Snap.
